src/RFC-0205_LedgerTransactions.md

```diff
@@ -0,0 +1,148 @@
+- [RFC-0203: Stealth Address](RFC-0203_StealthAddress.md)
+
+## Introduction
+
+Hardware wallets are secure physical devices used to protect crypto assets and funds by requiring user interaction from the device.
+These air-gapped devices are used to sign for transactions by keeping all the secrets from the transactions on the hardware device. If
+the host machines that run the wallets are compromised by malware the secrets from the transactions are still safe as they are not
+kept on the machine as with regular wallets. The devices are very low power and only feature a very limited processing power. MimbleWimble 
+has intensive crypto operations that are in some instances very slow or not able to run on the hardware wallet at all. This RFC describes a way to 
+get around these limitations to have fully functional secure hardware wallet integration. 
+
+## Background
+
+Vanilla [Mimblewimble] only has a single secret per [UTXO], the blinding factor (\\( k_i \\) ), the Tari protocol extends the [Mimblewimble] protocol to include scripting in the form of [TariScript]. This adds a second secret per [UTXO] called the script_key (\\( k_s \\) ). In practical terms this means that while vanilla [Mimblewimble] only requires that a wallet wishing to spend an [UTXO], prove knowledge of (\\( k_i \\) ) by producing the kernel signature, this is not sufficient for Tari. A Tari wallet must also prove knowledge of the script key (\\( k_s \\) ), by producing the script signature. 
+
+## Requirements
+
+To properly implement hardware wallets we need the following requirements to be met:
+* No UTXO can be spent without a user physically approving the transaction on the hardware wallet.
+* Users need to verify transaction properties when signing for transactions.
+* The user must be able to receive transactions without having to authorize them on the hardware wallet.
+
+## Implementation
+
+### Entities
+Normal transactions have only a single entity, the wallet which controls all secrets and transactions. But with hardware wallets, we need to define two distinct types:
+* Signer: This is the entity that keeps the secrets for the transactions and approves them, aka the hardware wallet.
+* Helper: This entity is the program that helps the signer construct the transaction, send it over the network and scan the network, aka wallet.
+
+### Process Overview
+By splitting the ownership of the [UTXO]'s secret by assigning knowledge of only the script key (\\( k_s \\) ) to the signer, we can lift much of the heavy cryptography like bulletproof creation to the helper device by exposing (\\( k_i \\) ) to it. By looking at how [one-sided-stealth] transactions are created, we can construct the script key in such a way that the helper can calculate the public script key, but cannot calculate the private script key.
+
+All [UTXO]s created will be created with a script `PushPubkey(K_S)`. The key (\\( k_s \\) ) is created as follows:
```

Comment:
I gather, from the discussion below, that you mean to say that we must update the protocol to deprecate the current `NOP` default script so that there's always a script key present?